src/test/java/Quiz/dao/QuizDaoImplTest.java

```diff
@@ -0,0 +1,86 @@
+package Quiz.dao;
+
+import Quiz.model.Quiz;
+import User.DBUser;
+import User.dao.IUserDAO;
+import User.dao.UserDAOImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import javax.ws.rs.NotFoundException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class QuizDaoImplTest {
+
+    private IQuizDAO quizDAO = new QuizDaoImpl();
+    private IUserDAO userDAO = new UserDAOImpl();
+
+    @BeforeEach
+    void setUp() {
+        //TODO: It's probably very wrong to use the DAO classes here to seed the data before testing the DAO class.
+        //But for now, it seems to solve the issues of dealing with auto incrementing id's between tests.
```

Comment:
How about prepopulating the database with test data manually from a postgresql client?